### ISSUE TITLE
Multiple Itemcounts of same char in tooltip

### ISSUE THREAD

**Issue by Babelee** [Dyaxler/Altoholic_Vanilla#3]
Hi,
i spent several hours trying to find a way to fix this issue, but i didnt come up with anything helpful.
My problem is, that my Tooltip shows Multiple instances of the item count  of some items (not all).

it cind of looks like this:

Silverleaf
Babelee:    11(Bags:10, Bank:1)
Total owned: 11

Babelee:    11(Bags:10, Bank:1)
Total owned: 11

Babelee:    11(Bags:10, Bank:1)
Total owned: 11

this is totally annoying, because it overlaps with other tooltip info.

i first was thinking that its because i installed multiple versions of altoholic before i found this one to work korrectly, but i dont know if that is really the case.
and many items are shown correctly once.
anyone have an idea what i can do?

Thanks a lot for ur work!

**Comment by Dyaxler** [comment]
I would search through the ..\SavedVariables\Altoholic.lua and search for item number: 1617 - https://classicdb.ch/?object=1617 - Delete all references to this number and save it. Do it while the game isn't running. Next time you login and scan that character it should record the correct data.

Or, since you used more than one version of Altoholic it's possible that you have some references in your savedvariables file that the addon is displaying in your tables. I know it's a pain in the arse but deleting the Altoholic.lua in your SavedVariables directory will clear out the bad data. The draw back is that you'd have to log back in each toon to repopulate your database.

**Comment by Babelee** [comment]
Hi, Thanks for replying

so what i did before reading your post, i deleted 
World of Warcraft 1.12\WTF\Account\-account name-\SavedVariables\Altoholic.lua and Altoholic.lua.bak
thinking that all Data is cleared then and Altoholic will reload and fix the problem...
but it didnt really fix the problem.
and it only seems to affect all herbs. the other items are displayed korrectly and only once.
is there any other save file that altoholic uses that i can delete?


Edit: so what i did now was to disable all addons with only altoholic running.
now the information is shown only two times (on some herbs 3 times) instead of one time.
i found out that when i activate Informant (with EnhTooltiop and Stubby) then the information is displayed 3 times (on some herbs 4 times).

So IMHO im guessing it is some sort of library problem... does informant and altoholic share any data?
it seems to me the easiest way would be to delete everything concerning altoholic but i dont know all the file locations...

**Comment by ngabor7** [comment]
I have the exact same issue. It seems it only affects herbs. I also got Informant/EnhTooltiop/Stubby installed. It displays the item count by character and the total item count information 3 times in the tooltip, but the the background of the tooltip is not changed in size enough to contain all the letters, so this extra text is not within the borders of the tooltip.

**Comment by Dyaxler** [comment]
...there are a number of issues like this one I'm encountering too. Remember... it's a "back port" from a later version of LUA so the tool tip code might not be handling the data quite right yet. I'm scrubbing through issues as I go but in the past several weeks I haven't had the time to really examine the smaller, more annoying bugs.

**Comment by ngabor7** [comment]
Hey, it's cool man. This is an awesome addon and now it works perfectly on vanilla too. I can keep track of everything across alts, it's magic, so much information. Thanks for the work! I appreciate it.

**Comment by Babelee** [comment]
Yes, Thanx Dyaxler for all the wortk you put into this addon. im not trying to complain! i am very Happy as well for this Addon to work so good on vanilla. i can manage. i just wanted to try to rule out, that this is a other problem on my end and try to identify the problem as well. on some herbs it shows the information like 7 or 8 times and on rare ocacions it shows it correctly only 1 time. the Problem i see  so far only on Herbs.

**Comment by Dyaxler** [comment]
I haven't level an "herb'er" yet. - getting there.

But thanks! :)
